live/tests/acceptance/j1-compare-answer-solution-qcm-test.js

```diff
@@ -126,8 +126,8 @@ describe('Acceptance | j1 - Comparer réponses et solutions pour un QCM |', func
     it('j1.4.1 QCM correcte et cochée', async function () {
 
       await visit(RESULT_URL);
-      expect($(CHECKBOX_CORRECT_AND_CHECKED)).to.have.lengthOf(0);
-      expect($(LABEL_CORRECT_AND_CHECKED)).to.have.lengthOf(0);
+      expect($(CHECKBOX_CORRECT_AND_CHECKED)).to.exist;
+      //expect($(LABEL_CORRECT_AND_CHECKED)).to.have.lengthOf(0);
```

Comment:
Je l'avais laissé en attendant ton avis sur l'utilisation de to.exist ou lenghtOh(0).
Je supprime le code mort !